src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -137,6 +137,16 @@ public static class Rule
         category: Category.CodeQuality,
         isEnabled: true);
 
+    public static DiagnosticDescriptor EmbedValidResourceFiles => New(
+        id: 2000,
+        title: "Embed valid resource files",
+        message: "Invalid XML: {0}",
+        description: "A resource file should contain valid XML.",
```

Comment:
Its not totally clear if it is about content or structure

Reply:
Because it is about both?!